sympy/series/fourier.py

```diff
@@ -170,6 +189,42 @@ def truncate(self, n=3):
 
         return Add(*terms)
 
+    def sigma_approximation(self, n=3):
+        """
+        Return sigma approximation of Fourier series with respect to order n.
+
+        Sigma approximation is a method allowing for reduction of Gibbs effect
+        in discontinuous function approximation with Fourier series partial sums.
```

Comment:
A mathematical definition (like the formula from the Wikipedia page; you can render it with `:math:`) would be useful here.